__tests__/app.test.js

```diff
@@ -0,0 +1,29 @@
+const app = require('../app');
+const request = require('supertest')
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js');
+const {articleData, commentData, topicData, userData} = require('../db/data/test-data/index.js');
+
+beforeEach(() => seed({ articleData, commentData, topicData, userData }));
+afterAll(() => db.end());
+
+describe('/api/topics', () => {
+    test('Request to an invalid endpoint receives a 404 error', () => {
+        return request(app)
+        .get('/api/forklift')
+        .expect(404)
+    })
```

Comment:
This is a really nice test to see, but it should not be in the describe block for /api/topics. It would also be nice to include a .then for this to ensure the correct message is in the response object.